dist/s_clang_format

```diff
@@ -97,7 +97,7 @@ fi
 for f in $search; do
     cat "$f" | \
             clang-format --fallback-style=none | \
-            python dist/s_comment.py > "$t" || exit 1
```

Comment:
Good catch but I believe this should be a different ticket.

Reply:
Good idea to make a new ticket.  That way we can look at and discuss all uses of "python" in dist (there are others).  I always run within a virtual environment, so my "python" is always "python3", but others may do this differently.  Just created WT-11737

Reply:
[WT-11737](https://jira.mongodb.org/browse/WT-11737) has been created for this purpose

Reply:
Thanks @ddanderson!